content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -18,19 +18,24 @@ Using the latest compatible version of kubectl helps avoid unforeseen issues.
 
 The following methods exist for installing kubectl on Windows:
 
-- [Install kubectl binary with curl on Windows](#install-kubectl-binary-with-curl-on-windows)
+- [Install kubectl binary on Windows (via direct download or curl)](#install-kubectl-binary-on-windows-via-direct-download-or-curl)
 - [Install on Windows using Chocolatey, Scoop, or winget](#install-nonstandard-package-tools)
 
-### Install kubectl binary with curl on Windows
+### Install kubectl binary on Windows (via direct download or curl)
 
-1. Download the latest {{< skew currentVersion >}} patch release:
-   [kubectl {{< skew currentPatchVersion >}}](https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe).
+1. You have two options for installing kubectl on your Windows device
 
-   Or if you have `curl` installed, use this command:
+   - Direct download:
+     
+     Download the latest {{< skew currentVersion >}} patch release binary directly for your specific architecture by visiting the [Kubernetes release page](https://kubernetes.io/releases/download/#binaries). Be sure to select the correct binary for your architecture (e.g., amd64, arm64, etc.).
+   
+   - Using curl:
 
-   ```powershell
-   curl.exe -LO "https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe"
-   ```
+     If you have `curl` installed, use this command:
+
+     ```powershell
+     curl.exe -LO "https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe"
+     ```
```

Comment:
(nit) Could also use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) here.

Reply:
Ah, I didn’t know about this feature. I’ll consider that next time, the PR has been merged.